### PR TITLE
[Doctest] Add configuration_roformer.py 

### PR DESCRIPTION
**Pull request by hegdeadithyak**
Adds configuration_roformer.py to utils/documentation_tests.txt

Based on https://github.com/huggingface/transformers/issues/19487

@ydshieh Please have a look at this 🌞

### CODE REVIEW

**Comment by Adithya4720** [comment]
@ydshieh Please have a look at this 🌞

File: utils/documentation_tests.txt
Comment (ydshieh) [inline utils/documentation_tests.txt:1]: This is file is no longer used. Could you remove this change? Thanks.

**Review by ydshieh — COMMENTED** [review]
Thank you for the PR @Adithya4720 .  I left a comment :-)

File: utils/documentation_tests.txt
Comment (Adithya4720) [reply utils/documentation_tests.txt:1]: Oh Sure!!!

**Comment by ydshieh** [comment]
Sorry, I forgot to check, but we should remove the entry `src/transformers/models/roformer/configuration_roformer.py` from the file `utils/not_doctested.txt`. Otherwise the doctest won't be triggered.

**Comment by Adithya4720** [comment]
@ydshieh Sir,Please Check this.

**Comment by Adithya4720** [comment]
@ydshieh Sir, Please Check this 😁

**Comment by ydshieh** [comment]
OK, you corrected it - I was somehow confused by the previous commit! Thanks

**Comment by Adithya4720** [comment]
Thankyou

**Review by ydshieh — APPROVED** [review]
Thanks again!

**Pull request merged**